integration-tests/bats/sql-local-remote.bats

```diff
@@ -21,6 +21,14 @@ teardown() {
     teardown_common
 }
 
+get_staged_tables() {
+    dolt status | awk '
+        match($0, /new table:\ */) { print substr($0, RSTART+RLENGTH) }
+        /Untracked tables:/ { exit }
+        /Tables with conflicting dolt_ignore patterns:/ { exit }
+    '
```

Comment:
This was designed to mirror the output of git status, which doesn't support structural output. I agree it would be nice. And trivial to implement with `git status` migrated. I created an issue for it.